docs/migration.md

```diff
@@ -0,0 +1,82 @@
+<!--[metadata]>
++++
+aliases = ["/engine/misc/deprecated/"]
```

Comment:
Yes, that will cause a redirect every time someone tries to hit the deprecation file.